app/extensions/safe/manageAuthNotifications.js

```diff
@@ -15,7 +18,61 @@ export const REQ_TYPES = {
     MDATA     : 'MDATA'
 };
 
-
+export const createAuthRequestElement = (authReqData) =>
+{
+  const R = React.createElement;
```

Comment:
@hunterlester i think it's cleaner if we stay using jsx here instead of creating the objects in this fashion. Then it's consistent across the project.

We could also then separate out the component from this file.